backend/app/Actions/Website/GetCurrentUserWebsiteAction.php

```diff
@@ -15,14 +15,14 @@ public function __construct(WebsiteRepository $repository)
         $this->repository = $repository;
     }
 
-    public function execute(int $websiteId): GetCurrentUserWebsiteResponse
+    public function execute(string $websiteId): GetCurrentUserWebsiteResponse
```

Comment:
Why did you do it? This method works with int, it means you have to cast any value to int before you pass to action

Reply:
It was an errors in tests